wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
+2. Plugin everything (Ethernet cable, audio gear, etc.)
+
+... to minimise CPU load and to ensure that no other application tries to use your sound card.
+
+You can have a look at [Hardware Setup](Hardware-Setup) for more information on how to set up your hardware. If you need help setting up ASIO4All on Windows, have a look at the [ASIO4All Setup guide on the Windows installation page](Installation-for-Windows#setting-up-asio4all).
+
+### Jamulus main window
+If you open Jamulus, you will see a window which looks like this:
+
+![Jamulus main window](https://user-images.githubusercontent.com/20726856/100796017-4cfa4500-3420-11eb-9d35-aa206d392a5a.png)
+
+### 1. Setting up a profile
+To allow others to identify who you are, you should set your profile. To do so, click on "View" (bar on the top)>"My Profile..."
+Now you will see this:
+
+![Musician Profile](https://user-images.githubusercontent.com/20726856/100796411-e590c500-3420-11eb-9b62-e50d02cb009b.png)
+
+Fill at least "Alias/Name" to set your name and close the window.
+
+### 2. Connecting to a server
+Now you can connect to a server. Click on "Connect" in the Jamulus main window, select a genre list (selection box on the top left), select a server, and click on "Connect". Now you should be connected to the server. If you have problems here, they are most likely due to wrongly configured audio gear or network issues. Have a look at the [Troubleshooting page](Client-Troubleshooting).
```

Comment:
Is it a "genre list"? Seems more descriptive to say just "select a genre" - or "select a genre from the list in the top left"?

Reply:
Agree. Fixed